packages/block-library/src/group/theme.scss

```diff
@@ -2,7 +2,5 @@
 	&.has-background {
 		// Matches paragraph Block padding
 		padding: $block-bg-padding--v $block-bg-padding--h;
-		margin-top: 0;
-		margin-bottom: 0;
```

Comment:
This was conflicting with the this consistent margin styling. @jasmussen maybe you know where in our code base we define the block margins and whether we should move these to classic.css now.